### PR TITLE
Created MaintainerCard component (and story)

### PR DESCRIPTION
**Pull request by Gammarkin**
## Describe your changes
Created MaintainerCard component, story, and made css for the component

## Screenshots - If Any (Optional)
image

## Link to issue
Create MaintainerCard component (and story) #170

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.
the button with the gh user has a anchor that should take the user to the gh link of the person on the card.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/172"></a>

### CODE REVIEW

File: components/MaintainerCard/MaintainerCard.module.css
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.module.css:19]: Please stick to rem, not a mix of em and rem

File: components/MaintainerCard/MaintainerCard.module.css
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.module.css:21]: These should not be needed

File: components/MaintainerCard/MaintainerCard.module.css
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.module.css:69]: There should not be a need for any media queries on a card that should look the same on all screen sizes.

File: components/MaintainerCard/MaintainerCard.module.css
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.module.css:32]:
```suggestion
    font-size: 1rem;
```

File: components/MaintainerCard/MaintainerCard.module.css
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.module.css:42]:
```suggestion
    font-size: 1rem;
```

File: components/MaintainerCard/MaintainerCard.tsx
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.tsx:4]: none of the props need to start with the word 'maintainer'. It is implied from the component.

File: components/MaintainerCard/MaintainerCard.tsx
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.tsx:35]: h1 is not the right element to use here. I'd suggest that it's not a heading at all so a p tag with styling is more than adequate.

File: components/MaintainerCard/MaintainerCard.tsx
Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.tsx:40]: A button nested inside a link is not good semantic html. This is a link to another site, therefore button should be removed and the link can be styled to look like a button instead.

Comment (EmmaDawsonDev) [inline components/MaintainerCard/MaintainerCard.tsx:40]: Also, please remove the target=_blank rel=noreferrer as links should not open in a new window/tab

File: stories/MaintainerCard.stories.tsx
Comment (EmmaDawsonDev) [inline stories/MaintainerCard.stories.tsx:27]: I don't think this is needed. The data can be placed directly in the args below.

File: stories/MaintainerCard.stories.tsx
Comment (EmmaDawsonDev) [inline stories/MaintainerCard.stories.tsx:29]:
```suggestion
export const MaintainerCard = Template.bind({})
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Apologies for so many comments! This is looking good but there's a few changes I'd like to see before it can be merged. Let me know if anything seems unclear.

**Comment by Gammarkin** [comment]
no problem! Thanks for the suggestions. Will work on it!

File: stories/MaintainerCard.stories.tsx
Comment (ctoffanin) [inline stories/MaintainerCard.stories.tsx:15]: This is not working. You renamed the component during import and the story is not showing up.

Comment (ctoffanin) [reply stories/MaintainerCard.stories.tsx:15]: My suggestion is to replace the content this file with this
```
import React from "react"
import { ComponentStory, ComponentMeta } from "@storybook/react"

import {
	MaintainerCard,
	MaintainerProps,
} from "../components/MaintainerCard/MaintainerCard"

export default {
	title: "Example/MaintainerCard",
	component: MaintainerCard,
} as ComponentMeta<typeof MaintainerCard>

const Template: ComponentStory<typeof MaintainerCard> = (args) => (
	<MaintainerCard {...args} />
)

export const DesktopMaintainerCard = Template.bind({})

DesktopMaintainerCard.args = {
	image:
		"https://img.freepik.com/free-photo/handsome-smiling-young-african-man_171337-9650.jpg",
	fullName: "John Doe",
	description:
		"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisl vel tincidunt luctus, nunc nisl aliquam massa, eget aliquam nisl nisl eu nunc. Sed euismod, nisl vel tincidunt luctus, nunc nisl aliquam massa, eget aliquam nisl nisl eu nunc.",
	githubLink: "https://github.com",
} as MaintainerProps
```

To keep the original component's name and avoid confusion with naming.

**Review by ctoffanin — CHANGES_REQUESTED** [review]

**Comment by ctoffanin** [comment]
Sorry for the delay. We had a crazy busy week.
Please check my comments and let me know if something is not clear, and I'll be back to you.

**Review by ctoffanin — APPROVED** [review]

**Comment by ctoffanin** [comment]
Nice, thank you so much for contributing!

**Pull request merged**
